packages/grafana-alerting/src/grafana/api/v0alpha1/mocks/handlers/handlers.ts

```diff
@@ -0,0 +1,17 @@
+import { createReceiverHandler } from './ReceiverHandlers/createReceiverHandler';
+import { deleteReceiverHandler } from './ReceiverHandlers/deleteReceiverHandler';
+import { deletecollectionReceiverHandler } from './ReceiverHandlers/deletecollectionReceiverHandler';
+import { getReceiverHandler } from './ReceiverHandlers/getReceiverHandler';
+import { listReceiverHandler } from './ReceiverHandlers/listReceiverHandler';
+import { replaceReceiverHandler } from './ReceiverHandlers/replaceReceiverHandler';
+import { updateReceiverHandler } from './ReceiverHandlers/updateReceiverHandler';
+
+export const handlers = [
+  listReceiverHandler(),
```

Comment:
Do all handlers return `undefined` by default? `data` is undefined and handler functions return `HttpResponse.json(data)`
If so, I'm a bit concerned about tests debugging. If a handler is not mocked at all and the test is trying to use it MSW will throw a warning. In this case debugging might be tricky because we do mock all endpoints but they reply with `undefined` leading to an unusual component behavior